daffodil-core/src/main/scala/org/apache/daffodil/grammar/GrammarTerm.scala

```diff
@@ -18,17 +18,13 @@
 package org.apache.daffodil.grammar
 
 import org.apache.daffodil.exceptions.Assert
-import org.apache.daffodil.processors.parsers.Parser
-import org.apache.daffodil.processors.unparsers.Unparser
-import org.apache.daffodil.dsom.SchemaComponent
 import org.apache.daffodil.oolag.OOLAG.OOLAGHostImpl
-import org.apache.daffodil.compiler.ParserOrUnparser
-import org.apache.daffodil.compiler.BothParserAndUnparser
+import org.apache.daffodil.compiler.{ParserOrUnparser, BothParserAndUnparser}
```

Comment:
revert these import changes. Intellij IDEA is doing this automatically due to incorrect settings.